src/NServiceBus.AcceptanceTests/Core/Routing/MessageDrivenSubscriptions/When_using_autosubscribe_with_missing_publisher_information.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace NServiceBus.AcceptanceTests.Core.Routing.MessageDrivenSubscriptions
+{
+    using AcceptanceTesting;
+    using AcceptanceTests.EndpointTemplates;
+    using Logging;
+    using NUnit.Framework;
+    using System.Linq;
+    using System.Threading.Tasks;
+
+    public class When_using_autosubscribe_with_missing_publisher_information : NServiceBusAcceptanceTest
```

Comment:
This wasn't a new test in the Core folder in #5192. Why not follow the same pattern and modify the existing test in the non-Core Routing folder?

Reply:
My bad, I’ll fix it